src/redux/reducers/samples/initialState.js

```diff
@@ -0,0 +1,26 @@
+const sampleTemplate = {
+  name: null,
+  uuid: null,
+  type: null,
+  species: null,
+  complete: false,
+  error: false,
+  fileNames: [],
+  files: {},
+};
+
+const sampleFileTemplate = {
+  objectKey: '',
+  name: null,
+  size: 0,
+  mime: '',
+  success: false,
+  error: false,
```

Comment:
Is `error` to store error messages? If so, maybe it's better to set it's default value to an empty string?

Reply:
> error : false if there is no error associated with sample. Else, contain array of error messages. Default : false

I just verified if an empty array was falsy, but it is not

Reply:
I agree with Xavier's choice here, having error to be false and a string if there is an error is actually the common practice in Redux and it also lends itself to easier coding styles.